include/seqan3/range/view/trim.hpp

```diff
@@ -244,7 +245,7 @@ namespace seqan3::view
  * ```
  */
 
-seqan3::detail::trim_fn const trim;
+inline constexpr auto trim = detail::deep_view_adaptor_wrapper{seqan3::detail::trim_fn{}};
```

Comment:
More a question of understanding: Why is this inline constexpr and for to_rank and others it is inline const?

Reply:
Ideally this would all be constexpr, but forsome views from range-v3 this doesn't work and the ones you mentioned wrap views from range-v3. From my understanding this might change for the final ranges in the standard library, but it's also not really important for us since we don't currently do range stuff at compile time.

I currently try to make these things `constexpr` whenever possible, but if it doesn't work, I just make them `const`.